lectures/textures/README.md

```diff
@@ -0,0 +1,342 @@
+
+---
+
+Taking all of this into account, we can now update `render()` to match our new setup:
+
+```c++
+void render() {
+    glClear(GL_COLOR_BUFFER_BIT);
+    
+    program.SetModelMatrix(model_matrix);
+    
+    // Vertices
+    float vertices[] = {
+        -0.5f, -0.5f, 0.5f, -0.5f, 0.5f, 0.5f,  // triangle 1
+        -0.5f, -0.5f, 0.5f, 0.5f, -0.5f, 0.5f   // triangle 2
+    };
+    
+    glVertexAttribPointer(program.get_position_attribute(), 2, GL_FLOAT, false, 0, vertices);
+    glEnableVertexAttribArray(program.get_position_attribute());
+
+    // Textures
+    float texture_coordinates[] = {
+        0.0f, 1.0f, 1.0f, 1.0f, 1.0f, 0.0f,     // triangle 1
+        0.0f, 1.0f, 1.0f, 0.0f, 0.0f, 0.0f,     // triangle 2
+    };
+    
+    glVertexAttribPointer(program.get_tex_coordinate_attribute(), 2, GL_FLOAT, false, 0, texture_coordinates);
+    glEnableVertexAttribArray(program.get_tex_coordinate_attribute());
+    
+    // Bind texture
+    glBindTexture(GL_TEXTURE_2D, g_player_texture_id);
+    glDrawArrays(GL_TRIANGLES, 0, 6); // we are now drawing 2 triangles, so we use 6 instead of 3
+    
+    // We disable two attribute arrays now
+    glDisableVertexAttribArray(program.get_position_attribute());
+    glDisableVertexAttribArray(program.get_tex_coordinate_attribute());
+    
+    SDL_GL_SwapWindow(display_window);
+}
+```
+
+<sub>**Code Block 4**: A `render()` function worthy of textures.</sub>
+
+### Part 4: _Blending_
+
+We're almost there, I promise.
+
+The last bit we have to worry about is blending. We can think about this as what happens when two textures overlap each other. We need to tell OpenGL how to react to situations like these.
+
+In OpenGL, blending is off by default, so we need to enable it so that our images are transparent. We only need to do this once, so it'll go in our `initialisation()` function. Then, we need to select one of the various blending functions available to us from OpenGL's libraries. We'll go for one of the most common ones, but you can look at OpenGL's official documentation for [**more**](https://learnopengl.com/Advanced-OpenGL/Blending) if you're curious.
+
+```c++
+// enable blending
+glEnable(GL_BLEND);
+glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
+```
+
+---
+
+If at this point, your images do not load, and you are on...
+
+- **Mac**: Go to "Build Phases" and add your image to the `Copy Files` area. Make sure `"Copy only when installing"` is unchecked!
+
+- **Windows**: Use the File Explorer to copy your images manually into your project's folder.
+
+### Part 5: _Drawing multiple objects_
+
+One of the most common errors I've seen students encounter so far comes when they try to create multiple objects on the screen. Here's a tip: the three lines that actually draw the object on the screen are the following, which exist in the `render()` part of our program:
+
+```c++
+program.SetModelMatrix(model_matrix);
+glBindTexture(GL_TEXTURE_2D, texture_id);
```

Comment:
Change to `g_` for both model matrix and texture ID (I'm pretty sure they're both global, at least)